src/freehand_selection.jl

```diff
@@ -0,0 +1,60 @@
+function init_freehand_select(ctx::ImageContext)
+    c = ctx.canvas
+    enabled = Signal(true)
+    dragging = Signal(false)
+
+    dummybtn = MouseButton{UserUnit}()
+
+    min_x = Signal(1.0)
```

Comment:
Slightly more general would be
```julia
min_x, max_x = map(x->Signal(Float64(x)), extrema(indices(ctx.image, 2)))
```
since not all images are guaranteed to start indexing at 1.